README.md

```diff
@@ -3,30 +3,70 @@
+This is the [PyTorch](https://github.com/pytorch/pytorch) version of the [OpenNMT](https://opennmt.net) project, an open-source (MIT) neural machine translation framework. It is designed to be research friendly to try out new ideas in translation, summary, morphology, and many other domains. Some companies have proven the code to be production ready.
 
 We love contributions. Please consult the Issues page for any [Contributions Welcome](https://github.com/OpenNMT/OpenNMT-py/issues?q=is%3Aissue+is%3Aopen+label%3A%22contributions+welcome%22) tagged post. 
 
 <center style="padding: 40px"><img width="70%" src="http://opennmt.github.io/simple-attn.png" /></center>
-
 Before raising an issue, make sure you read the requirements and the documentation examples.
 
 Unless there is a bug, please use the [Forum](http://forum.opennmt.net) or [Gitter](https://gitter.im/OpenNMT/OpenNMT-py) to ask questions.
 
+----
+
+# Announcement - OpenNMT-py 2.0
+
+**We're happy to announce the upcoming release v2.0 of OpenNMT-py.**
+
+The major idea behind this release is the -- almost -- complete **makeover of the data loading pipeline**. A new 'dynamic' paradigm is introduced, allowing to apply on the fly transforms to the data.
+
+This has a few advantages, amongst which:
+
+- remove or drastically reduce the preprocessing required to train a model;
+- increase the possibilities of data augmentation and manipulation through on-the fly transforms.
+
+These transforms can be specific tokenization methods, filters, noising, or any custom transform users may want to implement. Custom transform implementation is quite straightforward thanks to the existing base class and example implementations.
+
+You can check out how to use this new data loading pipeline in the updated [docs](https://opennmt.net/OpenNMT-py).
+
+All the readily available transforms are described [here](https://opennmt.net/OpenNMT-py/FAQ.html#what-are-the-readily-available-on-the-fly-data-transforms).
+
+### Performance
+
+Given sufficient CPU resources according to GPU computing power, most of the transforms should not slow the training down. (Note: for now, one producer process per GPU is spawned -- meaning you would ideally need 2N CPU threads for N GPUs).
+
+### Breaking changes
+
+For now, the new data loading paradigm does not support Audio, Video and Image inputs.
+
+A few features are also dropped, at least for now:
+
+- audio, image and video inputs;
+- source word features.
+
+For any user that still need these features, the previous codebase will be retained as `legacy` in a separate branch. It will no longer receive extensive development from the core team but PRs may still be accepted.
+
+Feel free to check it out and let us know what you think of the new paradigm!
+
+----
+
 
 Table of Contents
 =================
   * [Full Documentation](http://opennmt.net/OpenNMT-py/)
-  * [Requirements](#requirements)
+  * [Usage](#usage)
```

Comment:
Since all the text in this section is actually installing required packages, I prefer requirements in this case though.

Reply:
I'll go for "Setup" I think, because it's not only requirements, as we clone and install opennmt.